api/estados/saopaulo.js

```diff
@@ -1,652 +1,654 @@
-  "Santa Maria da Serra",
-  "Santa Mercedes",
-  "Santa Rita d'Oeste",
-  "Santa Rita do Passa Quatro",
-  "Santa Rosa de Viterbo",
-  "Santa Salete",
-  "Santana da Ponte Pensa",
-  "Santana de Parnaíba",
-  "Santo Anastácio",
-  "Santo André",
-  "Santo Antônio da Alegria",
-  "Santo Antônio de Posse",
-  "Santo Antônio do Aracanguá",
-  "Santo Antônio do Jardim",
-  "Santo Antônio do Pinhal",
-  "Santo Expedito",
-  "Santópolis do Aguapeí",
-  "Santos",
-  "São Bento do Sapucaí",
-  "São Bernardo do Campo",
-  "São Caetano do Sul",
-  "São Carlos",
-  "São Francisco",
-  "São João da Boa Vista",
-  "São João das Duas Pontes",
-  "São João de Iracema",
-  "São João do Pau d'Alho",
-  "São Joaquim da Barra",
-  "São José da Bela Vista",
-  "São José do Barreiro",
-  "São José do Rio Pardo",
-  "São José do Rio Preto",
-  "São José dos Campos",
-  "São Lourenço da Serra",
-  "São Luís do Paraitinga",
-  "São Manuel",
-  "São Miguel Arcanjo",
-  "São Paulo",
-  "São Pedro",
-  "São Pedro do Turvo",
-  "São Roque",
-  "São Sebastião",
-  "São Sebastião da Grama",
-  "São Simão",
-  "São Vicente",
-  "Sarapuí",
-  "Sarutaiá",
-  "Sebastianópolis do Sul",
-  "Serra Azul",
-  "Serra Negra",
-  "Serrana",
-  "Sertãozinho",
-  "Sete Barras",
-  "Severínia",
-  "Silveiras",
-  "Socorro",
-  "Sorocaba",
-  "Sud Mennucci",
-  "Sumaré",
-  "Suzanápolis",
-  "Suzano",
-  "Tabapuã",
-  "Tabatinga",
-  "Taboão da Serra",
-  "Taciba",
-  "Taguaí",
-  "Taiaçu",
-  "Taiúva",
-  "Tambaú",
-  "Tanabi",
-  "Tapiraí",
-  "Tapiratiba",
-  "Taquaral",
-  "Taquaritinga",
-  "Taquarituba",
-  "Taquarivaí",
-  "Tarabai",
-  "Tarumã",
-  "Tatuí",
-  "Taubaté",
-  "Tejupá",
-  "Teodoro Sampaio",
-  "Terra Roxa",
-  "Tietê",
-  "Timburi",
-  "Torre de Pedra",
-  "Torrinha",
-  "Trabiju",
-  "Tremembé",
-  "Três Fronteiras",
-  "Tuiuti",
-  "Tupã",
-  "Tupi Paulista",
-  "Turiúba",
-  "Turmalina",
-  "Ubarana",
-  "Ubatuba",
-  "Ubirajara",
-  "Uchoa",
-  "União Paulista",
-  "Urânia",
-  "Uru",
-  "Urupês",
-  "Valentim Gentil",
-  "Valinhos",
-  "Valparaíso",
-  "constgem",
-  "constgem Grande do Sul",
-  "constgem Grande Paulista",
-  "Várzea Paulista",
-  "Vera Cruz",
-  "Vinhedo",
-  "Viradouro",
-  "Vista Alegre do Alto",
-  "Vitória Brasil",
-  "Votorantim",
-  "Votuporanga",
-  "Zacarias"
-  ]
+const sp = {
+  state: 'São Paulo',
+  abbreviation: 'sp',
+  cities: [
+    'Adamantina',
+    'Adolfo',
+    'Aguaí',
+    'Águas da Prata',
+    'Águas de Lindóia',
+    'Águas de Santa Bárbara',
+    'Águas de São Pedro',
+    'Agudos',
+    'Alambari',
+    'Alfredo Marcondes',
+    'Altair',
+    'Altinópolis',
+    'Alto Alegre',
+    'Alumínio',
+    'Álconstes Florence',
+    'Álconstes Machado',
+    'Álconsto de Carvalho',
+    'Alvinlândia',
+    'Americana',
+    'Américo Brasiliense',
+    'Américo de Campos',
+    'Amparo',
+    'Analândia',
+    'Andradina',
+    'Angatuba',
+    'Anhembi',
+    'Anhumas',
+    'Aparecida',
+    "Aparecida d'Oeste",
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)